spec/models/event_admin_right_spec.rb

```diff
@@ -18,11 +18,13 @@
 RSpec.describe EventAdminRight, type: :model do
   describe 'GET #index' do
     it 'sets defaults for seclevel and type' do
-      chair = FactoryGirl.create(:chair)
-      trigger = FactoryGirl.create(:user)
-      target = FactoryGirl.create(:user)
+      chair = FactoryGirl.build_stubbed(:chair)
+      trigger = FactoryGirl.build_stubbed(:user)
+      target = FactoryGirl.build_stubbed(:user)
       event = EventAdminRight.create(trigger_id: trigger.id, target_id: target.id, chair_id: chair.id)
```

Comment:
The test fails at this point because `Key (chair_id)=(1012) is not present in table "chairs".`
Because 'build_stubbed' only builds fake objects you have to use *create*